editor/find_in_files.cpp

```diff
@@ -779,7 +779,19 @@ void FindInFilesPanel::_on_item_edited() {
 }
 
 void FindInFilesPanel::_on_finished() {
-	_status_label->set_text(TTR("Search complete"));
+	String results_text;
+	int result_count = _result_items.size();
+	int file_count = _file_items.size();
+
+	if (result_count == 1 && file_count == 1) {
+		results_text = vformat(TTR("%d match in %d file."), result_count, file_count);
+	} else if (result_count != 1 && file_count == 1) {
+		results_text = vformat(TTR("%d matches in %d file."), result_count, file_count);
+	} else {
+		results_text = vformat(TTR("%d matches in %d files."), result_count, file_count);
+	}
```

Comment:
That's a good use case to start using the newly implemented plurals support with `TTRN()`: #40443.

Reply:
On a second thought, this is probably not a good use case for that, because there are 2 numbers and 3 permutations between them. Not sure how TTRN would help here. I'd be able to check either `result_count` or `file_count`, but the first option would still be separate. Picking one over the other seems arbitrary, no?

Reply:
I'd need to think more about it, but let's keep it simple for now and possibly improve laters. We haven't used `TTRN` anywhere in the translations yet, there are several places which could benefit from it.